src/components/Navbar.js

```diff
@@ -1,19 +1,42 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
+import { Link, Outlet } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import userIcon from '../images/usericon.png';
+import styles from '../Style/Navbar.module.css';
+import navbarstyles from '../Style/Books.module.css';
 
-function Navbar() {
-  return (
-    <nav>
-      <ul>
-        <li>
-          <Link to="/books">Books</Link>
-        </li>
-        <li>
-          <Link to="/categories">Categories</Link>
-        </li>
-      </ul>
+const Navbar = ({
+  path,
+}) => (
+  <>
+    <nav className={styles.panelbg}>
+      <div className={styles.BookstoreCMS}>
+        Bookstore CMS
+      </div>
+      <div>
+        <Link className={navbarstyles.BOOKS} to="/Books">
+          <div style={{ color: `${path ? 'black' : 'rgb(165, 162, 162)'}` }}>
+            BOOKS
+          </div>
+        </Link>
+      </div>
+      <div>
+        <Link className={navbarstyles.BOOKS} to="/Categories">
+          <div style={{ color: `${path ? 'rgb(165, 162, 162)' : 'black'}` }}>
+            CATEGORIES
+          </div>
+        </Link>
+      </div>
+      <div>
```

Comment:
1. I would recommend using the `NavLink` instead of the `Link` component here. This will add a default active class which will help in highlighting the active link in the navbar just like in zeplin design.
2. I would recommend moving the user icon in the navbar to the right end of the navbar just like in zeplin design.

| Your design  | zeplin design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/224469628-ed8e918e-8df9-4cd3-8e6f-192e7ff6bbf1.png) | ![image](https://user-images.githubusercontent.com/33268927/224469648-4b58aa9e-6a41-4a99-94fb-0be798c65c36.png) |

Reply:
Thanks, Now, icon is magined to the right and Links are NavLinks